composer.json

```diff
@@ -15,11 +15,12 @@
             "role": "Developer"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
Not sure this is required (or can be avoided)

Reply:
yes, required for github to run tests with master. but this fine, because minimum-stability is "root-only", means projects which use this package are not affected.